lib/web_ui/dev/driver_version.yaml

```diff
@@ -1,4 +1,10 @@
-
+## Version to be picked by this PR. The LUCI will have multiple cipd packages
```

Comment:
> Version to be picked by this PR

Which PR is this referring to? PR is a Github-specific term. You can run driver tests without any PRs.

Reply:
The PR that is running. I'm changing it with: "Version currently in use." I didn't want to say it initially since, I want the test readers to understand at a given time, there will probably be multiple Chrome versions tests are using (during the upgrade times especially).

I think this is especially a critical information for the Engine Sheriff. If one sees an error related to the Chrome version.

Reply:
I changed the comments. I also added documentation to README. Hopefully more clear now.